crossfit/backend/torch/op/base.py

```diff
@@ -33,7 +33,8 @@ def __init__(
         max_mem: str = "16GB",
         sorted_data_loader: bool = True,
         model_output_col: Optional[str] = None,
-        pred_output_col: str = "preds",
+        model_output_cols: Optional[list] = None,
```

Comment:
Outside of backward compatibility is there a reason to prevent combining `model_output_col` and `model_output_cols`? If that's the to either of the params and make the behavior such that only one code is followed (it seems like code only uses `self.model_output_cols`, so we can deprecate `model_output_col`)

Reply:
Yeah, Its for backward compatibility. Let me add a deprecation warning and remote it in the next release.